### PR TITLE
Remove custom proguard rules

### PR DESCRIPTION
**Pull request by gokhanaliccii**
Remove proguard rules from sample app because onfido-sdk comes with own proguard rules.

### CODE REVIEW

**Review by sergiomarqmoura — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -54,4 +54,4 @@ dependencies {
     implementation('com.google.android.gms:play-services-base:11.8.0') {
```
Comment (sergiomarqmoura) [inline app/build.gradle]: You should also remove the reference from:
`proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'`

File: app/proguard-rules.pro [hunk 2]
```diff
@@ -14,48 +14,4 @@
 # class:
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 # public *;
-#}
```
Comment (sergiomarqmoura) [inline app/proguard-rules.pro:17]: You should also remove the reference from:
`proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'`

Comment (gokhanaliccii) [reply app/proguard-rules.pro:17]: IMO i shouldn't remove because that config is default when you create new project with android studio

**Review by sergiomarqmoura — APPROVED** [review]

**Review by rfreitas — APPROVED** [review]
lgtm 👍

**Pull request merged**